### DISCUSSION TITLE
Setup with Proton Mail?

### DISCUSSION THREAD

**Discussion by kknopper** [moralmunky/Home-Assistant-Mail-And-Packages#714] [Q&A]
Hello, I have a Proton Mail account, and was wondering if it can be setup with this integration.

I'm using the Proton Mail Bridge app (https://proton.me/mail/bridge), on a different machine on my local network. This creates a local IMAP and SMTP server. I've tried connecting to both via the integration and just get an `Errno 111: Connection refused` in the logs. 

Not sure if this is possible with proton, but was curious if anyone else has attempted this

**Comment by firstof9** [comment]
Based on what I read it only listens on `localhost` you can't use it actually on the network.

**Reply by NotExpectedYet** [reply]
You can but it uses a self signed certificate which when I connect this package to it I get connection refused with ssl errors.
